set.mm

```diff
@@ -462875,6 +462899,13 @@ Class abstractions (a.k.a. class builders)
   disjdifr $p |- ( ( B \ A ) i^i A ) = (/) $=
     ( cdif cin c0 incom disjdif eqtr3i ) ABACZDIADEAIFABGH $.
 
+  $( If a singleton is a subset of another, their members are equal.  Deduction
+     version.  (Contributed by NM, 28-May-2006.)  (Revised by Thierry Arnoux,
+     11-Apr-2024.) $)
+  snsssnd $p |- ( ( A e. V /\ { A } C_ { B } ) -> A = B ) $=
```

Comment:
This is not in deduction form...

Reply:
Indeed, I've renamed it to `snsssng` (compare with [`snsssn`](https://us.metamath.org/mpeuni/snsssn.html))